src/main/scala/chiseltest/formal/backends/Maltese.scala

```diff
@@ -110,6 +132,27 @@ private[chiseltest] object Maltese {
     }
   }
 
+  // Produces a vcd file based on the witness is @annos contains WriteVcdAnnotation
+  private def processWitness(
+    circuit:    ir.Circuit,
+    sysInfo:    SysInfo,
+    annos:      AnnotationSeq,
+    witness:    Witness,
+    modelUndef: Boolean,
+    targetDir:  Path,
+    vcdSuffix:  String
+  ) = {
+    val writeVcd = annos.contains(WriteVcdAnnotation)
+    if (writeVcd) {
+      val sim = new TransitionSystemSimulator(sysInfo.sys)
+      sim.run(witness, vcdFileName = Some((targetDir / s"${circuit.main}.${vcdSuffix}.vcd").toString))
```

Comment:
When the simulator is run, the `_resetActive` constraint is violated in the induction case as we start from an arbitrary step. The message `ERROR: Constraint #_resetActive was violated!` will be printed every time an induction check fails.  I suppose this ok, but I'm not sure we can prevent it from noticing the constraints are violated unless we remove the `_resetActive` constraint. Maybe it could be classed as something entirely different.